.github/workflows/cd.yml

```diff
@@ -9,7 +9,9 @@ on:
 
 jobs:
   maven-cd:
-    uses: jenkins-infra/github-reusable-workflows/.github/workflows/maven-cd.yml@v1
+    uses: lemeurherve/github-reusable-workflows/.github/workflows/maven-cd.yml@v1
+    with:
+      java-version: 17
```

Comment:
should probably change the plugin to java 17 as well to do a full test?

Reply:
Is something like that sufficient?

https://github.com/jenkinsci/flyway-api-plugin/pull/9/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R36-R37

Reply:
yes